cms/lib/xblock/upstream_sync_container.py

```diff
@@ -54,15 +54,15 @@ def sync_from_upstream_container(
     return upstream_children
 
 
-def fetch_customizable_fields_from_container(*, downstream: XBlock, user: User) -> None:
+def fetch_customizable_fields_from_container(*, downstream: XBlock) -> None:
     """
     Fetch upstream-defined value of customizable fields and save them on the downstream.
 
     The container version only retrieves values from *published* containers.
 
     Basically, this sets the value of "upstream_display_name" on the downstream block.
     """
-    upstream = lib_api.get_container(LibraryContainerLocator.from_string(downstream.upstream), user)
+    upstream = lib_api.get_container(LibraryContainerLocator.from_string(downstream.upstream))
```

Comment:
Was this even working before?! Thanks for fixing.

Reply:
Thank `mypy` :) It caught it when I changed `get_container` to require named parameters after the key.